admin/sql/updates/2019-07-09-add-recommendation-table.sql

```diff
@@ -7,7 +7,7 @@ CREATE TYPE cf_recording_type AS ENUM('top_artist', 'similar_artist');
 CREATE TABLE recommendation.cf_recording (
   id                  SERIAL, -- PK
   user_id             INTEGER NOT NULL, --FK to "user".id
-  recording_msid      UUID NOT NULL,
+  recording_mbid      UUID NOT NULL,
```

Comment:
has this schema change already been run in production? Either way, we shouldn't change this here, in case we want to apply all schema changes in the order of their creation.